src/openloco/envionment.cpp

```diff
@@ -158,12 +158,12 @@ namespace openloco::environment
         if (!fs::exists(result))
         {
 #ifndef _WIN32
-            result = find_similar_file(result);
-            if (result.empty())
+            if (auto similarResult = find_similar_file(result); similarResult.empty())
```

Comment:
Two statements in an `if`? Which one is actually being used for the condition?

Reply:
http://en.cppreference.com/w/cpp/language/if#If_Statements_with_Initializer